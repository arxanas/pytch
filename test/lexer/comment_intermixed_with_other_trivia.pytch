# This is a comment followed by a newline.

# And then another comment.
