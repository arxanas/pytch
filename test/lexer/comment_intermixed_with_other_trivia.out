leading comment '# This is a comment followed by a newline.\n'
leading newline '\n'
leading comment '# And then another comment.\n'
the end of the file
